src/main/kotlin/uk/gov/dluhc/emailnotifications/EmailClient.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.dluhc.emailnotifications
```

Comment:
I understand this is more a complaint about how VAC does it than anything reasonable - but I don't really like this being outside the registercheckerapi package. I see it isn't very specific to register checker but I feel like either it needs to live in the shared libraries or it is a sufficiently small amount of email sending set up code that it can fit in the registercheckerapi package.

Reply:
Yeah I copied the setup from the VAC api - do you think:
a) it's worth changing here?
b) it's worth changing in VAC api?

Reply:
Apparently this code does already exist in the shared-library: https://github.com/communitiesuk/eip-ero-libs-api/tree/main/email-client

Could you try using this, and if it works and is simple enough, then make the change on vac and print api?

Reply:
Done! I've got the same changes ready for the VAC api, but I'm not sure what ticket to put the changes against. I'll chat to you on Monday :)

Reply:
Excellent, thank you!